Django/weatherdetector/templates/index.html

```diff
@@ -35,7 +35,7 @@
   </center>
   <div class="row">
     <div class="col-md-6 col-md-offset-3">
-      <h1>city</h1>
+      <h1>{{city}}</h1>
```

Comment:
This takes an input from user and displays the name of the City they want to see

Reply:
confirmed changes